healthcheck/Makefile

```diff
@@ -0,0 +1,106 @@
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
```

Comment:
Would be great to sync this with the other makefiles in the repo.  That said I don't think it's a blocker.

Reply:
Added the same license header;-)